source/documentation/common/upgrade/snip-upgrade_considerations.adoc

```diff
@@ -2,4 +2,16 @@
 :_content-type: SNIPPET
 .Upgrade Considerations
 
-When planning to upgrade, see link:https://access.redhat.com/articles/5268351[Red Hat Virtualization 4.4 upgrade considerations and known issues].
+* When planning to upgrade, see link:https://access.redhat.com/articles/5268351[Red Hat Virtualization 4.4 upgrade considerations and known issues].
+
+* When upgrading from Open Virtual Network (OVN) and Open vSwitch (OvS) 2.11 to OVN 2021 and OvS 2.15, the process is transparent to the user as long as the following conditions are met:
+
+** The engine is upgraded first.
```

Comment:
"the {engine name} is upgraded first" if this applies both upstream and downstream, or "Manager" for downstream only